src/main/java/frc/robot/subsystems/Drive/DriveSubsystem.java

```diff
@@ -107,6 +109,10 @@ public DriveSubsystem() {
 
           currentPose = odometry.getPoseMeters(); // todo needs to take the position from vision
           pidController = new PIDController(Drive.PID.kP, Drive.PID.kI, Drive.PID.kD);
+          
+          //creates a pid controller for steering  
+          pidControllerRotation = new PIDController(PID.Steer.kP, PID.Steer.kI, PID.Steer.kD);
```

Comment:
dont use these constants they are for the steer of the modules themselves they are not the pid constants you need you need for the whole subsystem you need new ones